plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/TestHiveMetastoreMetadataQueriesAccessOperations.java

```diff
@@ -500,7 +501,7 @@ public void testSelectColumnsFilterByTableName(boolean allTablesViewsImplemented
                 // some lengthy explanatory comment why variable count
                 // TODO switch to assertMetastoreInvocations when GET_TABLE invocation count becomes deterministic
                 // TODO this is horribly lot, why, o why we do those redirect-related calls in ... even if redirects not enabled?????????!!?!1
```

Comment:
unrelated: Is this line still relevant after https://github.com/trinodb/trino/commit/7a1df276963a6482fe0a03953499adf2040a27b1 ?

cc @huberty89